unlock-app/src/components/lock/UnlockedFlag.js

```diff
@@ -0,0 +1,60 @@
+import styled from 'styled-components'
+import React from 'react'
+import { connect } from 'react-redux'
+import { RoundedLogo } from '../interface/Logo'
+import { Colophon } from './Overlay'
+import { mapStateToProps } from './ShowUnlessUserHasKeyToAnyLock'
```

Comment:
It feels strange to import this from another component?
I have no clear opinion on that but maybe worth extracting to a helper function in another PR?

Reply:
we do this in a couple of other places, I agree this would be cleaner.